receiver/splunkenterprisereceiver/client_test.go

```diff
@@ -34,11 +34,9 @@ func (m *mockHost) GetExtensions() map[component.ID]component.Component {
 
 func TestClientCreation(t *testing.T) {
 	cfg := &Config{
-		ClientConfig: confighttp.ClientConfig{
+		IdxEndpoint: confighttp.HTTPClientSettings{
```

Comment:
Probably need to revert that line?

Reply:
Oooh, sorry, did not see this change in 0.94. Nice catch!